docs/guide/save.rst

```diff
@@ -30,3 +30,28 @@ For example, to save an image with low quality:
 
     with open('low_quality.jpg', 'wb') as f:
         i.save_as_jpeg(f, quality=40)
+
+Progressive JPEGs
+-----------------
+
+:meth:`~Image.save_as_jpeg` takes a ``progressive`` keyword argument, that can
+be used to make it output a progressive JPEG file:
```

Comment:
Now I've seen that JPEGs are not progressive by default - we should clarify this in the docs.